docs/cryptodoc/src/05_09_kyber.rst

```diff
@@ -1,138 +1,299 @@
+   | Botan Function               | Code Reference                                                                              | Purpose                                                              | Algorithms of [FIPS-203]_ |
+   +==============================+=============================================================================================+======================================================================+===========================+
+   | ``encode_polynomial_vector`` | :srcref:`[src/lib/pubkey/kyber/kyber_common]/kyber_algos.cpp:184|encode_polynomial_vector`  | Byte encoding of polynomial vectors                                  | 5                         |
+   +------------------------------+---------------------------------------------------------------------------------------------+----------------------------------------------------------------------+---------------------------+
+   | ``decode_polynomial_vector`` | :srcref:`[src/lib/pubkey/kyber/kyber_common]/kyber_algos.cpp:192|decode_polynomial_vector`  | Byte decoding of polynomial vectors                                  | 6                         |
+   +------------------------------+---------------------------------------------------------------------------------------------+----------------------------------------------------------------------+---------------------------+
+   | ``polynomial_from_message``  | :srcref:`[src/lib/pubkey/kyber/kyber_common]/kyber_algos.cpp:204|polynomial_from_message`   | Byte decoding of the K-PKE message :math:`m`                         | 6                         |
+   +------------------------------+---------------------------------------------------------------------------------------------+----------------------------------------------------------------------+---------------------------+
+   | ``polynomial_to_message``    | :srcref:`[src/lib/pubkey/kyber/kyber_common]/kyber_algos.cpp:212|polynomial_to_message`     | Byte encoding of the K-PKE message :math:`m`                         | 5                         |
+   +------------------------------+---------------------------------------------------------------------------------------------+----------------------------------------------------------------------+---------------------------+
+   | ``expand_keypair``           | :srcref:`[src/lib/pubkey/kyber/kyber_common]/kyber_algos.cpp:321|expand_keypair`            | Create public and secret keys from the seeds :math:`d` and :math:`z` | 13, 16                    |
+   +------------------------------+---------------------------------------------------------------------------------------------+----------------------------------------------------------------------+---------------------------+
+   | ``compress_ciphertext``      | :srcref:`[src/lib/pubkey/kyber/kyber_common]/kyber_algos.cpp:352|compress_ciphertext`       | Compress, byte encode, and concatenate polynomial vector             | 5, Formula 4.7            |
```

Comment:
From just reading the document: Why is this now in `kyber_algos` and not, as from the preceding sections, in `kyber_helpers`?

Reply:
This method does all three things (compression, byte encoding, and concatenation) in one and uses the `kyber_helpers` internally.